lib/models/app/userPrefs.ts

```diff
@@ -36,4 +50,7 @@ export class UserPrefsMutator extends ItemMutator {
   setWebPref(key: WebPrefKey, value: any) {
     this.setAppDataItem(key as any, value);
   }
+  setMobilePref(key: MobilePrefKey, value: any) {
+    this.setAppDataItem(key as any, value);
```

Comment:
Since, for example, `sortBy` is present in both `MobilePrefKey` and `WebPrefKey`, wouldn't that mean setting that key on mobile would overwrite the web preference?

Reply:
Yes indeed. I need to use different keys then.

Reply:
If value clashing is something we have to avoid then I think we should merge `MobilePrefKey` and `WebPrefKey` to have the compiler make sure there is no overlap.

Reply:
I guess it's tradeoffs. I like the merge idea but also keeping them separate seems..cleaner?

Reply:
I agree that it makes things clearer, however merging the enums will prevent overlap bugs, since they will be caught at compile time. IMO this is the winning tradeoff but maybe there's another way to have compile-time safety

Reply:
What if you create an unused enum that is the intersection of both, that simply exists for type safety (if that is even possible)?

Reply:
I'm thinking and I don't think there's another decent way to model this…